tools/deployment/macos_packaging/PkgInfo

```diff
@@ -0,0 +1 @@
+APPL????
```

Comment:
Is this a binary file? If so, it should get added to the linguist diff suppression in `.gitattributes`

What is it?